www/include/Administration/parameters/DB-Func.php

```diff
@@ -693,6 +728,41 @@ function updateGeneralConfigData($gopt_id = null)
     $centreon->initOptGen($pearDB);
 }
 
+/**
+ * Update Resource Status environment variable in PHP file.
+ *
+ * @throws \Exception
+ */
+function updateResourceStatusMode(): void
```

Comment:
I'm not sure this is the best place for this function, IMO this file should only contains operation on database. and switching an environment variable is not appropriate here (maybe better in www/include/common/common-Func.php)